index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>My portfolio</title>
+</head>
+<body>
+    
+    <header>
+        <ul class="top mobile">
+            <li> <a href="#">9:15</a> </li>
+            <li> <a href="#"><img src="images/wifi.png" alt=""></a></li>        
+        </ul>
+        <ul class="top mobile">
+            <li class="welcome"> <a href="#"> welcome</a></li>
+            <li> <a href="#"> <img src="images/Union.svg" alt=""></a></li>
```

Comment:
- [ ] Put the hamburger button is on the right and has a nav tag as requested in the project requirements

Reply:
- [ ] This change has not been implemented as requested by the previous CR.